app/helpers/application_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module ApplicationHelper
+  def logged_in?
+    current_user
```

Comment:
If you define a method with a question mark (?) the idea is to return a true or false value but in this case, you are returning the `current_user` object so, please, change it accordingly.